spack/packages/dla-future-fortran/package.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# dlaf-no-license-check
+from spack import *
+
+
+class DlaFutureFortran(CMakePackage):
+    """
+    Fortran interface to the DLA-Future library.
+    """
+
+    homepage = "https://github.com/eth-cscs/DLA-Future-Fortran"
+    url      = "http://www.example.com/MyPackage-1.0.tar.gz"
```

Comment:
Yes. I was waiting for the first release since I'm not sure what the tag will end up being (i.e. `v0.1.0` or `0.1.0`). I'll probably do the former like in DLA-Future.